packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -105,19 +108,32 @@ export const MediaLibrary = () => {
 
   const assets = assetsData?.results;
   const assetCount = assets?.length ?? 0;
-
   const isNestedFolder = !!query?.folder;
+
   const isLoading =
     foldersLoading || folderStructureIsLoading || permissionsLoading || assetsLoading;
+  const isFolder = !isLoading && query.folder;
```

Comment:
What do you think about re-using `const isNestedFolder = !!query?.folder;` from above?